modules/dqt/php/view.class.inc

```diff
@@ -0,0 +1,162 @@
+<?php declare(strict_types=1);
+/**
+ * Handles queries for CouchDB.
```

Comment:
I don't understand what this class is supposed to do based on the comments. How does it handle queries? What queries does it handle? What is "the View class for requests"? It seems to be only for CSVs, but.. what does that have to do with 'view'?

Reply:
Hi @driusan, here is an old comment from @xlecours about why the class is named view.

> Good morning Alizée. Looking at https://github.com/aces/Loris/pull/5911 , here is my thought:
You are seeing the module endpoints as services as opposed to resources access/interaction. Instead of having a GetDocument and SaveDocument classes, a single Document class that implements get->GET and save->POST seems more appropriate. After that, a View class to access data from 'datadictionary', 'sessions', 'search' and 'instruments' using something like GET /dataquery/views/search?operator=equals&category=demographics&field=Sex&value=Female

Eventually, the goal would be to use the view class for other things a user would like to view. Please let me know if you think a better name exists or what I should do.

Reply:
@maltheism I am not sure it applies to this PR. Maybe I can clarify. By View, i meant the views in CouchDB (e.g.: datadictionary, sessions, search,  instruments, ...)

Reply:
Hi @xlecours do you know of a good name I could use here instead of View for the class name that's used for fetching the candidates & sessions depending on the user's imported csv?

Reply:
I don't think you need to help of the backend to do that. All that the dqt is doing is to manage the list of applicable {Candid, visit_label} pairs.

Reply:
Hi @xlecours, @driusan the PR is kind of old but I believe the backend code is needed.
Please see this [attachment.zip](https://github.com/aces/Loris/files/6672707/attachment.zip) for an example "CSV file" from the user that they quickly want to generate fields by importing the CSV file.

See screenshot:
<img width="629" alt="Screen Shot 2021-06-17 at 3 15 56 PM" src="https://user-images.githubusercontent.com/16293415/122459762-4deb6200-cf7f-11eb-926c-ea162f394271.png">

Here are the results needed from the backend depending on if PSCID or DCCID is selected as the radio button.

[example_from_server_dccids.txt](https://github.com/aces/Loris/files/6672725/example_from_server_dccids.txt)

[example_from_server_pscids.txt](https://github.com/aces/Loris/files/6672728/example_from_server_pscids.txt)

Using the data from the backend to then create all the fields as seen here:

<img width="947" alt="Screen Shot 2021-06-17 at 3 22 12 PM" src="https://user-images.githubusercontent.com/16293415/122460219-db2eb680-cf7f-11eb-8c13-101e97d5c81a.png">

I don't believe that can all be done on the frontend without querying CouchDB. Please let me know if I'm mistaken and if I'm not what name should the class be for this CSV feature Leigh wants added.

Reply:
Ok then. As for the classname, I propose `DemographicIdentifiersFilterBuilder`